README.md

```diff
@@ -215,116 +215,116 @@ have some inaccuracies in number parsing:
-|                           D/gdc |  2.283<sub>±0.028</sub> |  116.61<sub>±00.04</sub> + 600.39<sub>±00.00</sub> |  53.37<sub>±00.83</sub> |
-|                         Haskell |  2.291<sub>±0.051</sub> |      4.44<sub>±00.05</sub> + 4.68<sub>±00.03</sub> |  43.10<sub>±02.05</sub> |
-|                            Ruby |  2.426<sub>±0.058</sub> |  124.00<sub>±00.02</sub> + 410.69<sub>±00.01</sub> |  57.63<sub>±01.08</sub> |
-|                     Ruby (YAJL) |  2.443<sub>±0.047</sub> |  123.96<sub>±00.02</sub> + 283.34<sub>±00.00</sub> |  44.94<sub>±01.47</sub> |
-|                 Scala (uPickle) |  2.516<sub>±0.053</sub> |  284.73<sub>±00.09</sub> + 762.67<sub>±23.36</sub> |  59.39<sub>±04.12</sub> |
-| F#/.NET Core (System.Text.Json) |  2.577<sub>±0.083</sub> |  485.37<sub>±00.05</sub> + 453.88<sub>±02.45</sub> |  55.03<sub>±05.61</sub> |
-|                          D/ldc2 |  2.585<sub>±0.057</sub> |  112.92<sub>±00.06</sub> + 680.09<sub>±00.04</sub> |  53.34<sub>±01.70</sub> |
-|                    Ruby (--jit) |  2.589<sub>±0.022</sub> |  124.01<sub>±00.06</sub> + 410.80<sub>±00.04</sub> |  50.43<sub>±00.51</sub> |
-|                       Rust (jq) |  3.584<sub>±0.035</sub> |  113.83<sub>±00.02</sub> + 777.86<sub>±00.90</sub> |  81.39<sub>±00.50</sub> |
-|                      Ruby/jruby |  3.876<sub>±0.084</sub> | 453.42<sub>±10.48</sub> + 1412.22<sub>±28.09</sub> | 108.53<sub>±06.77</sub> |
-|    C++/g++ (Boost.PropertyTree) |  4.483<sub>±0.118</sub> | 112.95<sub>±00.04</sub> + 1440.06<sub>±00.00</sub> | 101.16<sub>±06.49</sub> |
-|                           D/dmd |  5.098<sub>±0.075</sub> |  113.65<sub>±00.06</sub> + 680.25<sub>±00.01</sub> | 102.46<sub>±04.22</sub> |
-|                        Vala/gcc |  5.473<sub>±0.217</sub> |      0.00<sub>±00.00</sub> + 0.00<sub>±00.00</sub> |  99.96<sub>±06.82</sub> |
-|                      Vala/clang |  5.602<sub>±0.056</sub> |      0.00<sub>±00.00</sub> + 0.00<sub>±00.00</sub> | 101.12<sub>±00.60</sub> |
-|               Perl (JSON::Tiny) | 12.605<sub>±0.279</sub> |  125.30<sub>±00.07</sub> + 528.63<sub>±00.10</sub> | 231.12<sub>±05.85</sub> |
-|        Ruby/truffleruby (--jvm) | 16.962<sub>±0.464</sub> | 721.72<sub>±12.93</sub> + 1648.53<sub>±20.81</sub> | 479.86<sub>±14.66</sub> |
-|                Ruby/truffleruby | 40.369<sub>±0.806</sub> | 738.03<sub>±01.52</sub> + 2269.36<sub>±70.04</sub> | 834.73<sub>±65.49</sub> |
+|                        Language |                 Time, s |                                        Memory, MiB |                Energy, J |
+| :------------------------------ | ----------------------: | -------------------------------------------------: | -----------------------: |
+| C++/g++ (DAW JSON Link NoCheck) |  0.095<sub>±0.004</sub> |    112.69<sub>±00.01</sub> + 0.00<sub>±00.00</sub> |    2.60<sub>±00.21</sub> |
+|    C++/g++ (simdjson On-Demand) |  0.098<sub>±0.002</sub> |   113.27<sub>±00.03</sub> + 59.81<sub>±00.00</sub> |    2.14<sub>±00.05</sub> |
```

Comment:
this is updated to master libraries, or the same?

Reply:
It's always using the latest master branch as I do the full clean before maintenance updates.